packages/wallets/src/components/Base/ModalStepWrapper/__tests__/ModalStepWrapper.spec.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
```

Comment:
minor: probably should also tests for children rendering,

Reply:
Updated.